rero_ils/modules/holdings/api_views.py

```diff
@@ -43,3 +44,38 @@ def holding_availability(holding_pid):
     return jsonify({
         'availability': holding.available
     })
+
+
+@api_blueprint.route('/<holding_pid>/pattern/preview/', methods=['GET'])
```

Comment:
I would use:
GET /api/holdings/1/patterns/preview?size=10
and
POST /api/holding/patterns/preview?size=20

or
GET /api/holdings/patterns/preview/1?size=10
and
POST /api/holdings/patterns/preview?size=10